src/cloud-api-adaptor/versions.yaml

```diff
@@ -30,8 +30,8 @@ git:
     url: https://github.com/confidential-containers/guest-components
     reference: 277617af60c32661819c1132ffbf3db8dc6e1b9f
   kata-containers:
-    url: https://github.com/kata-containers/kata-containers
-    reference: 3.5.0
+    url: https://github.com/mkulke/kata-containers
+    reference: abc704a72034248e08e60b9b3a1309b575a95014
```

Comment:
Are you intending to merge with this fork? Is there anything beyond https://github.com/kata-containers/kata-containers/pull/9811 needed?

Reply:
no, i'd wait til the kata PR is merged and change it here.

Reply:
Great - I'll not wait for the power static checks to be sorted and merge it now :)